tests/testthat/test_all_open.R

```diff
@@ -0,0 +1,23 @@
+library(testthat)
+context("all open intervals\n")
+library(iregnet)
+
+set.seed(1)
+toy.features <- matrix(rnorm(10), 5, 2)
+toy.targets <- rbind(
+  c(1, 2),
+  c(-Inf, 5),
+  c(1, Inf),
+  c(-Inf, 3),
+  c(-Inf, 4))
+
+test_that("iregnet works with at least one closed interval", {
+  fit <- iregnet(toy.features, toy.targets)
+  expect_true(inherits(fit, "iregnet"))
+})
+
+test_that("iregnet works for all open intervals", {
+  fit <- iregnet(toy.features[-1,], toy.targets[-1,])
+  expect_true(inherits(fit, "iregnet"))
```

Comment:
Should unregularized solution exist for this problem? `Survreg` does not converge with this random data either:
```
set.seed(1)
toy.features <- matrix(rnorm(10), 5, 2)
toy.targets <- rbind(
  c(1, 2),
  c(-Inf, 5),
  c(1, Inf),
  c(-Inf, 3),
  c(-Inf, 4))

toy.targets[toy.targets == Inf] = NA
toy.targets[toy.targets == -Inf] = NA
t <- Surv(toy.targets[, 1], toy.targets[, 2], type='interval2')
fit_s <- survreg(t[-1, ] ~ toy.features[-1,])
```